src/peer_digest.cc

```diff
@@ -140,7 +146,6 @@ peerDigestNeeded(PeerDigest * pd)
 static time_t
 peerDigestIncDelay(const PeerDigest * pd)
 {
-    assert(pd);
```

Comment:
If it is still accurate, let's keep this assertion. Otherwise, we have more problems to solve because this code dereferences `pd`.